data/wanderer/wanderers start.txt

```diff
@@ -562,7 +511,7 @@ mission "Wanderers: Defend Vara Ke'sok"
 			branch hint
 				has "Wanderers: Defend Vara Ke'sok Hint: done"
 			`Soon after you land in the spaceport, there is a flurry of activity as many of the Wanderers fly to their ships, which begin to take off from the planet. In the sky, you see bright flashes that may be distant weapons fire or explosions.`
-			`	You are buffeted by wind as one of the Wanderers lands next to your ship and says, in the Hai language, "Unfettered [raider, pirate] fleet comes. We have put food on freighter, hope they take [offering, gift] and leave. If not [appeased, satisfied] you help convince them to leave. Yes?"`
+			`	You are buffeted by wind as one of the Wanderers lands next to your ship and says, in their language, "Unfettered [raider, pirate] fleet comes. We have put food on freighter, hope they take [offering, gift] and leave. If not [appeased, satisfied] you help convince them to leave. Yes?"`
```

Comment:
```suggestion
			`	You are buffeted by wind as one of the Wanderers lands next to your ship and says, "Unfettered [raider, pirate] fleet comes. We have put food on freighter, hope they take [offering, gift] and leave. If not [appeased, satisfied] you help convince them to leave. Yes?"`
```
It might be easier to just get rid of that phrase.